mobile/cmp/multiZoneGrid/Types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2023 Extremely Heavy Industries Inc.
+ */
+
+import {PersistOptions} from '@xh/hoist/core';
+
+export type Zone = 'tl' | 'tr' | 'bl' | 'br';
+
+export interface ZoneMapping {
+    /** Field to display. Must match a Field found in the Store */
+    field: string;
+    /** True to prefix the field value with its name */
+    showLabel?: boolean;
+}
+
+export interface ZoneLimit {
+    /** Min number of fields that should be mapped to the zone */
+    min?: number;
+    /** Max number of fields that should be mapped to the zone */
+    max?: number;
+    /** Array of allowed fields for the zone */
+    only?: string[];
```

Comment:
`allowedFields` ?

Reply:
Think its a better name, and happy to change, but wanted to point out that this name (and 'limits') were taken from the Sencha API.